src/embedWebFonts.ts

```diff
@@ -141,7 +141,9 @@ async function getCSSRules(
                         rule,
                         rule.startsWith('@import')
                           ? (importIndex += 1)
-                          : sheet.hasOwnProperty('cssRules').length,
+                          : // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                            // @ts-ignore
+                            sheet.hasOwnProperty('cssRules').length,
```

Comment:
Is there any other solution to this? `hasOwnProperty` returns boolean so `.length` is not a valid method

Reply:
can we use `('cssRules' in sheet).length`for this?